qm.8.md

```diff
@@ -0,0 +1,121 @@
+% QM 8
+
+## NAME
+QM - Setup Containerized environment for running Functional Safety QM (Quality Management) software.
+
+## SYNOPSIS
+This package allow users to setup an environment which prevents applications
+and container tools from interfering with other all other processes on the
+system.
+
+The QM runs its own version of systemd and Podman to isolate not only the
+applications and containers launched by systemd and Podman but systemd and
+Podman themselves.
+
+Software installed into the QM environment under the /usr/lib/qm/rootfs
+directory is automatically isolated from the host. If developers need to
+further isolate their applications from other processes in the QM they should
+use container tools like Podman.
+
+## INSTALL
+
+After you have installed the software package, you need to execute the
+/usr/share/qm/setup script to setup and install the /usr/lib/qm/rootfs
+with packages to run an isolated environment. By default the script installs
+the selinux-policy-targeted, podman, systemd, hirte packages and then enables
+and runs a qm.container Podman quadlet.
```

Comment:
Is what you have right?  It seems like a connector is missing between qm.container and the Podman quadlet.  But I'm showing my QM ignorance here.  Maybe?
```suggestion
and runs a Podman quadlet qm.container.
```